src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -3718,10 +3716,8 @@ private string ConvertAttestationProtocolToString(SqlConnectionAttestationProtoc
                 case SqlConnectionAttestationProtocol.HGS:
                     return "HGS";
 
-#if ENCLAVE_SIMULATOR
-                case SqlConnectionAttestationProtocol.SIM:
-                    return "SIM";
-#endif
+                case SqlConnectionAttestationProtocol.None:
+                    return "None";
```

Comment:
NIT: This is similar method to the one in Crypto EnclaveDelegate.Crypto.cs, returning enum values and can be removed.

Reply:
Done.